client/document/index.jsx

```diff
@@ -72,6 +73,11 @@ class Document extends Component {
 			( initialReduxState
 				? `var initialReduxState = ${ jsonStringifyForHtml( initialReduxState ) };\n`
 				: '' ) +
+			( dehydratedReactQueryState
```

Comment:
Similar to the redux store, this dehydrates the `react-query` state and will be used on the client side.